packages/angular/src/generators/component-story/component-story.spec.ts

```diff
@@ -0,0 +1,87 @@
+import type { Tree } from '@nrwl/devkit';
+import * as devkit from '@nrwl/devkit';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import * as storybookUtils from '../utils/storybook';
+import { componentStoryGenerator } from './component-story';
+
+describe('componentStory generator', () => {
+  let tree: Tree;
+  const libName = 'ng-lib1';
+  const storyFile = `libs/${libName}/src/lib/test-button/test-button.component.stories.ts`;
+
+  beforeEach(async () => {
+    tree = createTreeWithEmptyWorkspace();
+
+    const libGenerator = wrapAngularDevkitSchematic('@nrwl/angular', 'library');
+    const componentGenerator = wrapAngularDevkitSchematic(
+      '@schematics/angular',
+      'component'
+    );
+
+    await libGenerator(tree, { name: libName });
```

Comment:
This will need to be updated now likely as we have the `library` generator merged.